demux/demux_playlist.c

```diff
@@ -270,9 +271,73 @@ static bool scan_dir(struct pl_parser *p, char *path,
     return true;
 }
 
+/* http://www.davekoelle.com/alphanum.html */
```

Comment:
License violation.

Reply:
Isn't this MIT licensed?

Reply:
The answer is that the actual copyright notice must be preserved as per MIT.